src/app/layout.tsx

```diff
@@ -13,10 +13,6 @@ BigInt.prototype.toJSON = function (): string {
 };
 
 export const metadata = {
-  metadataBase: new URL("https://vote.optimism.io"),
```

Comment:
You don't like my meta? ? 
Canonical is a good tag to have in there. Is there a problem with it?

Reply:
Oh this is unintentional. Missed that this was part of this commit as well

Reply:
sorry -- putting back